share/revng/test/configuration/revng-qa/for-runtime-abi-tests.yml

```diff
@@ -68,28 +68,61 @@ tags:
     variables:
       ABI_NAME: Microsoft_x86_cdecl
       GCC_CFLAGS:
-        - -DABIDEF=__attribute__((ms_abi,cdecl))
+        # safeguard in case somebody tries using it with gcc
```

Comment:
1. `s/safeguard/Safeguard/`
2. Maybe we could factor these out in a new tag `disable-for-gcc`? Probably not worth it, just an idea.

Reply:
1. Done.
2. That would feel weird. Especially considering that we also have those we don't want to put through MSVC.